### PR TITLE
Added feature for user to change password

### PR DESCRIPTION
**Pull request by siddnikh**
Solves #2459 
Note: Won't work for admin users

### CODE REVIEW

**Comment by nihal467** [comment]
test approved

File: src/Components/Users/UserProfile.tsx [hunk 1]
```diff
@@ -581,7 +784,7 @@ export default function UserProfile() {
             Check for software updates
             <p className="mt-1 text-sm leading-5 text-gray-600">
               Click the update button to see if you have the latest
-              &quot;care&quot; version.
+              &quotcare&quot version.
```
Comment (khavinshankar) [inline src/Components/Users/UserProfile.tsx]: why was this line changed?
image

**Review by khavinshankar — CHANGES_REQUESTED** [review]
If it was changed due to the formatting, please do double check the changes

File: src/Components/Users/UserProfile.tsx (hunk 1 above)
Comment (siddnikh) [reply src/Components/Users/UserProfile.tsx]: yeah, I've fixed that.

**Review by khavinshankar — APPROVED** [review]
